adapter/thing_management.md

```diff
@@ -70,26 +70,28 @@ Every adapter should define its own service name.
+| out  | evt.thing.exclusion_status_report | string     |                                                                        | Returns stage of the ongoing exclusion process,  see [`exclusion_status`](#definitions).                                    |
+| out  | evt.thing.exclusion_report        | object     |                                                                        | Reports thing exclusion, see [`thing_identifier`](#definitions).                                                            |
+| in   | cmd.thing.delete                  | str_map    |                                                                        | Requests thing deletion if device is offline or exclusion process is not supported, see [`thing_identifier`](#definitions). |
 
 ### Interface properties
 
-| Name                   | Example   | Required | Description                                                                  |
-|------------------------|-----------|----------|------------------------------------------------------------------------------|
-| `pin`                  | `"12345"` | No       | PIN to be used during the inclusion process.                                 |
-| `force_non_secure`     | `"true"`  | No       | Force non-secure inclusion process to pair the device.                       |
-| `supports_pin_request` | `"true"`  | No       | Indicates that the client supports PIN request during the inclusion process. |
+| Name                   | Example                                                                  | Required | Description                                                                            |
+|------------------------|--------------------------------------------------------------------------|----------|----------------------------------------------------------------------------------------|
+| `pin`                  | `"12345"`                                                                | No       | PIN to be used during the inclusion process.                                           |
+| `force_non_secure`     | `"true"`                                                                 | No       | Force non-secure inclusion process to pair the device.                                 |
+| `supports_pin_request` | `"true"`                                                                 | No       | Indicates that the client supports PIN request during the inclusion process.           |
+| `qr_code`              | `"385cfbfffee34acc,565977,06,0523,3248799657aee175c79ccdca394be8867460"` | No       | QR code data scanned from device, which contains Zigbee install code and IEEE address. |
+| `qr_ctx`               | `"zg_XXX"`                                                               | No       | Additional data required during Zigbee device pairing.                                 |
```

Comment:
...required during device pairing allowing to properly parse data provided in `code` property. 

As example put a real used code, maybe "zg_gd_spc"